es/ae/index.js

```diff
@@ -39,7 +39,7 @@ import { AE_AMOUNT_FORMATS } from '../utils/amount-formatter'
  * @param {String} tx - Transaction
  * @param {Object} [options={}] options - Options
  * @param {Object} [options.verify] verify - Verify transaction before broadcast, throw error if not valid
- * @return {String|String} Transaction or transaction hash
+ * @return {Object} Transaction
  */
 async function send (tx, options = {}) {
   const opt = R.merge(this.Ae.defaults, options)
```

Comment:
Can the usage of Ramda be replaced here the same way as in:

https://github.com/aeternity/aepp-sdk-js/blob/3d00de7e29c7271012ae9975aabbcf57056a82c4/es/ae/index.js#L69

Reply:
Sure, but I didn't reach that code yet. The `R.merge` is deprecated so, I'm planning to update it later.